Makefile

```diff
@@ -21,19 +21,13 @@ help:
 	@echo "build-base    - (re)build base docker image"
 	@echo "docs          - generate Sphinx HTML documentation, including API docs"
 
-.docker-build-base:
-	${MAKE} build-base
-
 .docker-build:
 	${MAKE} build
 
-build-base:
-	${DOCKERCOMPOSE} -f docker-compose.build.yml build --pull base
-	-rm -f .docker-build
-	touch .docker-build-base
-
 build: .docker-build-base
```

Comment:
You need to remove the `.docker-build-base` bit for this rule. Pretty sure that's why the tests are failing.